cnv/cnv_install/installing-container-native-virtualization.adoc

```diff
@@ -7,12 +7,12 @@ toc::[]
 Install {CNVProductName} to add virtualization functionality to your {product-title}
 cluster.
 
-You can use the {product-title} 4.2
+You can use the {product-title} {product-version}
```

Comment:
@ousleyp @aburdenthehand would it be ok to replace this here for 4.3?

Reply:
@vikram-redhat I think we can definitely replace those with {product-version}, yes. Or we could even remove the version entirely, if you think that makes more sense.

On a related note, I think we do need to make sure that the CNV version attribute is updated and used throughout the docs. I believe there are some hard-coded `2.1`s. (This is on my to-do list, so I can take care of it.) cc @aburdenthehand

Reply:
Thanks @ousleyp.